fixposition_driver_ros2:
  ros__parameters:
    fp_output:
      formats: [
                "ODOMETRY", "LLH", "ODOMENU", "ODOMSH", "ODOMSTATUS", "RAWIMU", "CORRIMU", 
                "IMUBIAS", "GNSSANT", "GNSSCORR", "EOE", "TEXT", "TF", "TP", # FP_A
                "GPGGA", "GPGLL", "GNGSA", "GPGST", "GPHDT", "GPRMC", "GPVTG", "GPZDA", "GXGSV" # NMEA
               ]
      type: "tcp"
      port: "21000"
      ip: "10.0.2.1" # change to VRTK2's IP address in the network
      rate: 200
      reconnect_delay: 5.0 # wait time in [s] until retry connection
      qos_type: "sensor_short" # Supported types: "sensor_short", "sensor_long", "default_short", "default_long"
      cov_warning: false
      nav2_mode: false

    customer_input:
      speed_topic: "/fixposition/speed"
      rtcm_topic: "/rtcm"
